public/js/upload.js

```diff
@@ -7,11 +7,10 @@ function init() {
 function handleFileSelect(event) {
     if(!event.target.files) return;
     const files = event.target.files;
-    const selectedList = document.querySelector('#selectedFiles ul');
-
-    Array.from(files).map(function (file) {
-        var li = document.createElement('li');
-        li.appendChild(document.createTextNode(file.name));
-        selectedList.appendChild(li);
-    });
+    const selected = document.querySelector("#selectedFiles ul");
+    selected.innerHTML = "";
+    for(var i=0; i<files.length; i++) {
+        const file = files[i];
+        selected.innerHTML += "<li>" + file.name + "<br/>" + "</li>";
```

Comment:
Yes it would.

`appendChild` and no string concatenation or appending to `innerHTML` would be preferred, however, as appending to `innerHTML` causes a DOM and layout rebuild where as `appendChild` would just cause a layout change.